disassemblers/ofrak_ghidra/ofrak_ghidra/components/ghidra_decompilation_analyzer.py

```diff
@@ -0,0 +1,78 @@
+from typing import Tuple
+from json.decoder import JSONDecodeError
+import os
+import re
+
+
+from ofrak.resource import Resource
+from ofrak.core.complex_block import ComplexBlock
+from ofrak.core.decompilation import DecompilationAnalysis, DecompilationAnalyzer
+from ofrak_ghidra.constants import CORE_OFRAK_GHIDRA_SCRIPTS
+from ofrak_ghidra.ghidra_model import OfrakGhidraMixin, OfrakGhidraScript
+
+
+def take_delimited(s: str, delim: str) -> Tuple[str, str]:
+    result = ""
+
+    if delim in s:
```

Comment:
Maybe add a descriptive comment here as to why you have to parse the output like this.